shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1765,6 +1765,15 @@ void WebContents::OnGetDefaultPrinter(
 
   base::string16 printer_name =
       device_name.empty() ? default_printer : device_name;
+
+  // We set the default to the system's default printer and only update
+  // if at the Chromium level if the user overrides.
+  // If there are no valid printers available on the network, we bail.
+  if (printer_name.empty() || !IsDeviceNameValid(printer_name)) {
+    std::move(print_callback).Run(false, "no valid printers available");
```

Comment:
`print_callback` might be `null` if user does not pass an callback, should check `if (print_callback)` here.